src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -64,7 +65,9 @@ private ConversationStore(PersistentStorageAgent persistentStorageAgent) {
     conversations = new ArrayList<>();
   }
 
-/** Access the current set of conversations known to the application. */
+/** Access the current set of conversations known to the application.
+ *  Have to get rid of the Groups because they are erroneously being categorized as Conversations
```

Comment:
Can you explain this a little more?

Reply:
You mean the comment? Yeah, for the longest time, since I was trying to make Group a subclass of Conversation, all the Groups were being typecast back to instances of Conversation. This is because of how subclasses work (I learned). But I fixed that so I'll delete the comment.